libs/Microsoft.MixedReality.WebRTC.Native/src/uwp/Microsoft.MixedReality.WebRTC.Native.UWP.vcxproj

```diff
@@ -104,7 +104,7 @@
       <CompileAsWinRT>false</CompileAsWinRT>
       <WarningLevel>Level4</WarningLevel>
       <TreatWarningAsError>true</TreatWarningAsError>
-      <AdditionalIncludeDirectories>..\..\include;..\;$(WebRTCCoreRepoPath);$(WebRTCCoreRepoPath)out\$(Configuration)_$(PlatformTarget)\gen;$(WebRTCCoreRepoPath)third_party\abseil-cpp;$(WebRTCCoreRepoPath)third_party\libyuv\include;$(WebRTCCoreRepoPath)out\$(Configuration)_$(PlatformTarget)\gen\protoc_out;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\..\include;..\;$(WebRTCCoreRepoPath);$(WebRTCCoreRepoPath)out\$(Configuration)_$(PlatformTarget)_uwp\gen;$(WebRTCCoreRepoPath)third_party\abseil-cpp;$(WebRTCCoreRepoPath)third_party\libyuv\include;$(WebRTCCoreRepoPath)out\$(Configuration)_$(PlatformTarget)_uwp\gen\protoc_out;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
nit: I was thinking to switch the build path to something like `out\(win|winuwp)\$(PlatformTarget)\$(Configuration)` so it's a bit more "standard"

Reply:
Good point, that even better. I will change to that.

Reply:
Fixed.